README.md

```diff
@@ -122,6 +122,14 @@ Override border color
 <Avvvatars value="best_user@gmail.com" borderColor="#fff" />
 ```
 
+### `borderRadius?: number`
+
+Override border radius
```

Comment:
Change it as follows:
"Override the radius of the avatar, it takes `size` by default to always turn it to a circle"